examples/android-ndk/build.gradle

```diff
@@ -0,0 +1,43 @@
+buildscript {
+    repositories {
+        google()
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath Libs.AGP
+        classpath "io.sentry:sentry-android-gradle-plugin"
+    }
+}
+
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion = 30
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 30
+        versionCode = 1
+        versionName = "1.0"
+    }
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+    }
+    ndkVersion "21.4.7075529"
```

Comment:
any reason to specify the version? otherwise, AGP chooses the default version based on the AGP versioning https://developer.android.com/studio/projects/install-ndk#default-ndk-per-agp

Reply:
Yup my bad. I was playing with a couple of versions I had installed locally and i forgot to clean it up.